src/assertion.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+function expect($value, $predicate) {
```

Comment:
@mathiasverraes, I hope you don't mind me asking, shouldn't `$predicate` be `callable` ?